mpa/modules/utils/task_adapt.py

```diff
@@ -139,3 +139,13 @@ def sync_transpose_tensor(src_w, dst_w):
         return src_w.t()
     else:
         raise ValueError('the size of model and checkpoint file are not matched')
+
+
+def get_dataset_without_wrapping(dataset):
```

Comment:
How about 'unwrap_dataset'

Reply:
I think the current function name looks better. What do you think?

Reply:
I think unwrap_dataset is more intuitive